image_classification/tensorflow/official/resnet/resnet_run_loop.py

```diff
@@ -263,10 +278,16 @@ def exclude_batch_norm(name):
 
     learning_rate = learning_rate_fn(global_step)
 
+    log_id = mlperf_log.resnet_print(key=mlperf_log.OPT_LR, deferred=True)
+    learning_rate = tf_mlperf_log.log_deferred(op=learning_rate, log_id=log_id,
+                                               every_n=100)
+
     # Create a tensor named learning_rate for logging purposes
     tf.identity(learning_rate, name='learning_rate')
     tf.summary.scalar('learning_rate', learning_rate)
 
+    mlperf_log.resnet_print(key=mlperf_log.OPT_NAME, value=mlperf_log.SGD)
```

Comment:
Maybe have a different SGD and Momentum constant. Or at least comment SGD is Momentum

Reply:
Done.